src/panels/config/users/ha-config-users.ts

```diff
@@ -43,36 +43,73 @@ export class HaConfigUsers extends LitElement {
           ),
           sortable: true,
           filterable: true,
+          width: "25%",
           direction: "asc",
           grows: true,
           template: (name) => html`
             ${name ||
             this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
           `,
         },
+        username: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.username"
+          ),
+          sortable: true,
+          filterable: true,
+          width: "20%",
+          direction: "asc",
+          grows: true,
+          template: (username) => html`
+            ${username ||
+            this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
+          `,
+        },
         group_ids: {
           title: this.hass.localize(
             "ui.panel.config.users.picker.headers.group"
           ),
           sortable: true,
           filterable: true,
-          width: "30%",
+          width: "20%",
           template: (groupIds) => html`
             ${this.hass.localize(`groups.${groupIds[0]}`)}
           `,
         },
+        is_active: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.is_active"
+          ),
+          type: "icon",
+          width: "10%",
+          sortable: true,
+          filterable: true,
+          template: (is_active) => html`
+            ${is_active ? html` <ha-icon icon="hass:check"> </ha-icon> ` : ""}
+          `,
+        },
+        is_owner: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.is_owner"
+          ),
+          type: "icon",
+          width: "10%",
+          sortable: true,
+          filterable: true,
+          template: (is_owner) => html`
+            ${is_owner ? html` <ha-icon icon="hass:check"> </ha-icon> ` : ""}
+          `,
+        },
         system_generated: {
           title: this.hass.localize(
             "ui.panel.config.users.picker.headers.system"
           ),
           type: "icon",
-          width: "80px",
+          width: "15%",
```

Comment:
```suggestion
```

Reply:
Without those widths, the columns become unusable, since the header is just a single letter.
![image](https://user-images.githubusercontent.com/114137/99830387-eb9fbf80-2b5d-11eb-99a9-e2ff851ce240.png)

Reply:
Then set them to fixed widths, no need to have them grow.